app/globalTeardown.ts

```diff
@@ -4,8 +4,11 @@ import { consoleDebug } from './utils';
 
 
 export default async () => {
-  const mainDataSource = getMainDataSource();
-  consoleDebug(`globalTeardown - wasEverInitialized: ${mainDataSource.wasEverInitialized()}`);
-  await mainDataSource.closeAndDelete_templateDb();
+  let  mainDataSource = getMainDataSource();
+  mainDataSource.createOrConnectToExistingTemplateDB();
```

Comment:
That's not the time to create the template DB for the first time.
Can't we just connect to it if it exists?

Reply:
Corrected : https://github.com/Asaf-S/jest-test-clean-db/pull/3